tests/tests_pytorch/core/test_datamodules.py

```diff
@@ -265,75 +265,6 @@ def test_full_loop(tmpdir):
     assert result[0]["test_acc"] > 0.6
 
 
-@pytest.mark.parametrize(
```

Comment:
removed this since this is almost the same as the one we have in `models/test_hooks.py::test_apply_batch_transfer_handler`

and we have unit tests for DataHookSelector which checks from which component the hooks is taken from.